jlc_kicad_tools/generate_jlc_files.py

```diff
@@ -110,7 +112,21 @@ def main():
 	cpl_output_path = os.path.join(opts.output_dir, project_name + "_cpl_jlc.csv")
 
 	db = ReadDB(opts.database)
-	if GenerateBOM(netlist_path, bom_output_path, opts) and FixRotations(cpl_path, cpl_output_path, db):
+	
+	# Parse the NETLIST looking for custom fields named JLCPCBRotation (case sensitive)
+	fixed_rotation_db = {}
+	tree = ET.parse(netlist_path)
+	root = tree.getroot()	
+	for component in root.findall('components/comp'):
+		JLCPCBRotation = component.find('./fields/field[@name="JLCPCBRotation"]')
+		if JLCPCBRotation!=None:
+			angle =float(JLCPCBRotation.text)
```

Comment:
space after '='

Reply:
I created these files using a plain text editor, do you have a lint tool or auto formatting tool you can apply consistent formatting to?  Python isn't a language I use very much or have the formatting tools for it.

Reply:
I personally don't use one as I also don't use Python often outside of work, but I imagine if you want a linter pylint is the obvious choice. I have been using Google's style guide just because that's what I'm used to, but it's almost identical to PEP-8, which is the most common style.

https://pep8.org/